example/src/scenarios/cross-device-flow-with-rp.tsx

```diff
@@ -80,7 +90,7 @@ export default async () => {
     ).then((t) => RP.getRequestObject(t, authRequestUrl, entity));
 
     // Attest Relying Party trust
-    // TODO [SIW-354]
+    await verifyTrustChain(trustAnchorEntity, requestObj.header.trust_chain);
```

Comment:
what do you think about adding this check also into the SDK function (like `getEntityConfiguration()`)making sure that the caller is not responsible for doing the checks.

Reply:
Interesting point. That is true for every trustable document we have (including PID, WIA, RequestObject, etc).

I think it's good, anyway I'd do it in another PR as it involves a sensible refactor of classes because:
* we must provide Trust Anchor data 
* we must model better error to avoid to hide informations on error.

Reply:
yep! Thanks you